apps/aechannel/test/aesc_fsm_SUITE.erl

```diff
@@ -3745,7 +3773,7 @@ channel_spec(Cfg, XOpts) ->
              , minimum_depth        => config(minimum_depth_factor, Cfg, ?MINIMUM_DEPTH_FACTOR)
              , client               => self()
              , noise                => [{noise, Proto}]
-             , timeouts             => #{idle => 20000}
+             , timeouts             => #{idle => 200000}
```

Comment:
Actual fix. This bumps the `open` state timer 10 times so the FSM will not die in the middle of the test